lineage/StateDistribution.py

```diff
@@ -31,8 +31,8 @@ def pdf(self, tuple_of_obs):  # user has to define how to calculate the likeliho
         # distribution observations), so the likelihood of observing the multivariate observation is just the product of
         # the individual observation likelihoods.
 
-        bern_ll = sp.bernoulli.pmf(k=tuple_of_obs[0], p=self.bern_p)  # bernoulli likelihood
-        gamma_ll = sp.gamma.pdf(x=tuple_of_obs[1], a=self.gamma_a, scale=self.gamma_scale)  # gamma likelihood
+        bern_ll = self.bern_p**(tuple_of_obs[0]) * (1.0 - self.bern_p)**(tuple_of_obs[0])
```

Comment:
This is wrong. You need to check whether `tuple_of_obs[0]` is 1 or 0. Right now it's the same whether it's 0 or 1.

Reply:
My bad, told myself to change this but forgot. I think the second exponent needs to be `1-tuple_of_obs[0]`.